packages/components/addon/components/hds/stepper/indicator/step.hbs

```diff
@@ -1,6 +1,6 @@
 <div class={{this.classNames}} ...attributes role="listitem" aria-label={{@status}}>
-  <div class="hds-stepper-indicator-step__svg-hexagon" aria-hidden="true">
-    <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+  <div class="hds-stepper-indicator-step__svg-hexagon">
+    <svg aria-hidden="true" width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Agree with this change and, as mentioned [in a previous comment](https://github.com/hashicorp/design-system/pull/470#pullrequestreview-1051030518), I suggest we move the class to the svg element and remove the div wrapper.

Reply:
Good catch, thank you! Fixed!